### PR TITLE
3 More Paper Friendly Font Colors are added & Contact Form's Style is improved to Align with Overall Design.

### PR DESCRIPTION
**Pull request by nijin26**
For users, there was a limited option to choose font colours that match papers. According to my research, colours like "Orange", "Violet" & "Dodgerblue" are also a good choice for users. Also, pink is the least used one since it is lighter and hard to read.

In Contact Page, the Experience Form's design didn't match the overall design. I mean the Input Boxes had sharp edges even though they get curved when focussed. 
But It would be better to have a small border-radius at the beginning and more border-radius on focus.

I am a GSSOC's Participant.

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
improve the pink font

**Comment by nijin26** [comment]
The pink colour is darkened by adding 'deep pink' without any change on the Display name.

Color's display name's first letter is capitalized for making it more readable and professional.

'dodgerblue' is added in the name of "Light Blue" where normal blue is also there.
'dark violet is added in the name of 'Violet"

**Comment by nijin26** [comment]
beforenow

All Colors:
[colors](https://imgur.com/z1ExixI)

File: src/pages/Editor/sections/Settings/Settings.js [hunk 1]
```diff
@@ -33,29 +46,62 @@ const Settings = () => {
...
 
           <div className={styles.vSeparator}></div>
 
           <label className={styles.downloadBtn} htmlFor="import">
             Import File
           </label>
-          <input id="import" style={{ display: "none" }} type="file" onChange={editContext.importTxt}></input>
+          <input
+            id="import"
+            style={{ display: "none" }}
```
Comment (smaranjitghose) [inline src/pages/Editor/sections/Settings/Settings.js:101]: Please avoid the styling of the code. If I merge this, it would cause merge conflict for all other PRs. Revert these changes. Otherwise the PR is fine!

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: src/pages/Editor/sections/Settings/Settings.js (hunk 1 above)
Comment (nijin26) [reply src/pages/Editor/sections/Settings/Settings.js:101]: oh ok,
 It's got auto-formatted by Prettier.

Comment (nijin26) [reply src/pages/Editor/sections/Settings/Settings.js]: Code style or format issue fixed 🙂

**Review by smaranjitghose — APPROVED** [review]
